api/flask_bot_server.py

```diff
@@ -1,70 +0,0 @@
-from __future__ import absolute_import
-from flask import Flask, request, jsonify
-import os
-import sys
-import json
-from typing import Any, Dict, Mapping, Union, List
-from werkzeug.exceptions import BadRequest
-from six.moves.configparser import SafeConfigParser
```

Comment:
Flask depends on werkzeug, so it is automatically installed at the time of installation. The goal here was to return a bad request response. We can either use this or define our own exception class and intialise and return it.